directory-listing/common.css

```diff
@@ -17,14 +17,37 @@ body {
   border-bottom: 0.5rem solid #607d8b;
 }
 
+h1{
+  width: 95%;
+  margin: 0 auto;
+  margin-top: 1rem;
+  font-family: monospace;
+}
+
+table {
+  text-align: left;
+  white-space: nowrap;
+  border-collapse: collapse;
+  width: 95%;
```

Comment:
Percentages don't scale well, instead of changing `width` for `table` and `h1`, use fixed `padding` values. This also allows you to remove all the `margin` values.

Reply:
What unit would you suggest using instead of percentages? rem?